itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestJdbcWithMiniLlapArrow.java

```diff
@@ -123,8 +123,8 @@ public static void afterTest() {
     return new LlapArrowRowInputFormat(Long.MAX_VALUE);
   }
 
-  // Currently MAP type is not supported. Add it back when Arrow 1.0 is released.
-  // See: SPARK-21187
+  // Currently, loading from a text file gives errors with Map dataType.
+  // This needs to be fixed when adding support for non-ORC writes (text and parquet) for the llap-ext-client.
```

Comment:
This comment doesn't make sense here as this test is not specific to llap-ext-client. Instead mention the Hive JIRA which would solve this issue.

Reply:
Ideally, anything that works for ORC should work for non-ORC as well. Pls enable the tests on Map data type and see if it works after Arrow upgrade.